lib/money_accessor.rb

```diff
@@ -5,19 +5,24 @@ def self.included(base)
 
   module ClassMethods
     def money_accessor(*columns)
+      variable_get = self <= Struct ? :[]  : :instance_variable_get
+      variable_set = self <= Struct ? :[]= : :instance_variable_set
```

Comment:
I know it's unlikely, but what if we include that in a class that defines `:<=`? Can you use `is_a?` instead? It's also more readable imo.